src/app/components/privacy-policy/privacy-policy.component.spec.ts

```diff
@@ -37,10 +41,33 @@ describe('PrivacyPolicyComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(PrivacyPolicyComponent);
     component = fixture.componentInstance;
+    de = fixture.debugElement;
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have equal number of section-title elements as nav elements', () => {
+
+    const ALL_NAV = de.queryAll(By.css('.privacy-nav'));
+    const ALL_TARGET = de.queryAll(By.css('.privacy-section-title'));
+
+    expect(ALL_NAV.length).toBeGreaterThan(0);
+    expect(ALL_TARGET.length).toBeGreaterThan(0);
+    expect(ALL_TARGET.length).toBe(ALL_NAV.length);
+  });
+
+  it('should have same section title as nav element title', () => {
+
+    const ALL_NAV = de.queryAll(By.css('.privacy-nav'));
```

Comment:
Line 54, 55 is sameas line 64,65. Can you apply [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) concept here?